src/config.ts

```diff
@@ -11,3 +11,13 @@ export const fallbackImages = {
 };
 
 export const REDIS_CHANGELOG_KEY = 'boot:latest_changelog';
+
+export enum StorageTopic {
+  Boot = 'boot',
+}
+
+export const generateStorageKey = (
+  topic: StorageTopic,
+  key: string,
+  identifier = 'global',
+): string => `${topic}:${key}:${identifier}`;
```

Comment:
We have a function to generate storage keys in the `apps` repo, so I've thought of doing the same here.